spec/drag-and-drop.spec.js

```diff
@@ -19,14 +19,6 @@ describe('Drag and Drop TestCase', function () {
             expect(editor.elements[0].className).toContain('medium-editor-dragover');
         });
 
-        it('should add medium-editor-dragover class even when data is invalid', function () {
```

Comment:
@nmielnik what was the thoughts behind this behavior at the first place? 
wouldn't be better to not do drag and drop at all when no files type is allowed? https://github.com/yabwe/medium-editor/issues/966